lib/class-wp-rest-menu-items-controller.php

```diff
@@ -261,7 +310,8 @@ protected function prepare_item_for_database( $request ) {
 				'menu-item-description' => $menu_item_obj->description,
 				'menu-item-attr-title'  => $menu_item_obj->attr_title,
 				'menu-item-target'      => $menu_item_obj->target,
-				'menu-item-classes'     => implode( ' ', $menu_item_obj->classes ), // stored in the database as array.
+				// stored in the database as array.
```

Comment:
I think this is `stored in the database as a string` right?

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/50cb8a8b16b8821c7d666c022872e07d4dcf9a11